Libraries/Components/Touchable/TouchableNativeFeedback.android.js

```diff
@@ -185,7 +185,10 @@ var TouchableNativeFeedback = React.createClass({
 
   _handleResponderMove: function(e) {
     this.touchableHandleResponderMove(e);
-    this._dispatchHotspotUpdate(e.nativeEvent.pageX, e.nativeEvent.pageY);
+    var [elementWidth, elementHeight] = (this.pressInLocation)
+    ? [this.pressInLocation.pageX - e.nativeEvent.locationX, this.pressInLocation.pageY - e.nativeEvent.locationY]
+    : [0, 0];
+    this._dispatchHotspotUpdate(e.nativeEvent.pageX - elementWidth, e.nativeEvent.pageY - elementHeight);
```

Comment:
Good point. I'll add a comment :)

Actually the bug also appears on the x coord but most of the time ripple elements will be about 90-100% width of the screen or small. In these cases the wrong placement is hard to determine.